src/cpptools.ts

```diff
@@ -284,7 +289,7 @@ export class CppConfigurationProvider implements cpt.CustomConfigurationProvider
     const {standard, extraDefinitions} = parseCompileFlags(flags, lang);
     const defines = (fileGroup.defines || target.defines).concat(extraDefinitions);
     const includePath = fileGroup.includePath ? fileGroup.includePath.map(p => p.path) : target.includePath;
-    const normalizedIncludePath = includePath.map(p => util.platformNormalizePath(p));
+    const normalizedIncludePath = includePath.map(p => util.lightNormalizePath(p));
```

Comment:
It was buggy. Causing duplicates with different casing. What consistency are you referring to?

Reply:
Are you saying I should fix the paths that are not filtered via platformNormalizePath? e.g. previously it would have ../Folder/.. and ../folder/.. paths.

Reply:
I don't think the shipping bits don't have the `platformNormalize` fix in them yet.

Reply:
Yeah, that's what I figured. However, it looks like we need to not change the cases in this particular case, so I'm adding more comments to explain why.

Reply:
> Oh, yeah, there's a problem with vscode.workspace.getWorkspaceFolder returning undefined when the path is lowercased on Windows

Wouldn't that be a bug for vscode?  I don't think we can guarantee that CMake will return correctly-cased paths.  And vscode shouldn't fail a path comparison based on incorrect casing on Windows.